### PR TITLE
update

### PR DESCRIPTION
**Pull request by drifted-mdl**

### CODE REVIEW

File: random/link.js [hunk 1]
```diff
@@ -4,22 +4,23 @@ import { rndArray } from 'core/math/rnd.js'
...
-  'raid', 'group', 'bikergirl', 'cacadoo', 'owl', 'sparrow', 'crocodile', 'god', 'ant', 'lion', 'treasure',
-  'glasses', 'crown', 'swamp', 'forest', 'breakfast', 'meal', 'shaman', 'winter', 'summer', 'spring']
+  'raid', 'group', 'bikergirl', 'cacadoo', 'owl', 'sparrow', 'crocodile', 'god', 'ant', 'lion', 'treasure', 'gem',
+  'glasses', 'crown', 'swamp', 'forest', 'breakfast', 'meal', 'shaman', 'winter', 'summer', 'spring', 'gold']
 
 const goals = ['slaying', 'wisdom', 'destruction', 'death', 'healing', 'luck', 'misfortune', 'rage', 'joy',
   'leveling', 'vengance', 'resurrection', 'mastery', 'loss', 'flames', 'fire', 'frost', 'glory', 'war', 'sprinting',
   'doom', 'love', 'archery', 'hunting', 'peace', 'failure', 'success', 'victory', 'denial', 'agility', 'despair',
-  'woodworking', 'cooking', 'herbalism', 'leatherworking', 'skincare', 'treasures', 'fury', 'trading']
+  'woodworking', 'cooking', 'herbalism', 'leatherworking', 'skincare', 'treasures', 'fury', 'trading', 'legacy']
```
Comment (zbot473) [inline random/link.js]: Skincare?? Lol

Comment (dekdevy) [reply random/link.js]: "of legacy" doesnt seem to make that much sense, i would move it to a noun. 
"glorious legacy of skincare" makes sense
"smelly dog of legacy", not sure if it makes sense grammatically.

File: random/name.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ const syllables = [ 'heim', 'tuk', 'ta', 'lor', 'bok', 'le', 'me', 'ka', 'na', '
   'stone', 'soul', 'pain', 'glib', 'glob', 'myth', 'drakar', 'breaker', 'lust', 'tack', 'stopper', 'hink', 'hunk', 'rot',
   'corpse', 'ash', 'fang', 'skin', 'cold', 'immer', 'ice', 'doctor', 'meph', 'isto', 'dia', 'blo', 'baal', 'battle', 'bubble',
   'bat', 'crack', 'drool', 'stink', 'beard', 'grim', 'dry', 'shave', 'hate', 'love', 'scum', 'master', 'crusher', 'spawn',
-  'meat', 'face']
+  'meat', 'face', 'silver', 'claw', 'dal', 'meteor', 'sword', 'myr']
```
Comment (dekdevy) [inline random/name.js]: try to keep it to 2 or less syllables (remove meteor). rest looks good!

**Comment by dekdevy** [comment]
these are awesome. lets remove forsaken (3syl) and then its good to go :)

**Pull request merged**